whipper/common/program.py

```diff
@@ -447,6 +451,10 @@ def getTagList(self, number, mbdiscid):
                 tags['MUSICBRAINZ_ALBUMID'] = mbidRelease
                 tags['MUSICBRAINZ_RELEASEGROUPID'] = mbidReleaseGroup
                 tags['MUSICBRAINZ_ALBUMARTISTID'] = mbidReleaseArtist
+                if len(composers) > 0:
+                    tags['COMPOSER'] = composers
+                if len(performers) > 0:
+                    tags['PERFORMER'] = performers
```

Comment:
Tiiiiiiny nitpick, but I’d prefer this code to be after the `WORKID` code. Just so that all the MBIDs are "listed" together. :) (Sorry I didn’t notice this earlier!)

Reply:
No problem :D